src/main/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class PastRecruiters(models.Model):
+    company_order_no = models.PositiveIntegerField(default=64)
+    company_name = models.CharField(max_length=50)
+    company_logo = models.ImageField(upload_to='media/company-logo', blank=True, null=True)
```

Comment:
Just remove `media/` for now, when we later add `media` in settings, this will automatically be fixed